backend/app/services/project_service.py

```
@@ -49,7 +49,11 @@ def create_project(
     # Get the max sort_order for this user's projects
     max_sort_order = (
         db.query(func.max(Project.sort_order))
-        .filter(Project.user_id == user_id, Project.is_active == True)
+        .filter(
+            Project.user_id == user_id,
+            Project.client_origin == project_data.client_origin,
+            Project.is_active == True,
```

Ruff (0.15.14)

[error] Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)